package.json

```diff
@@ -49,25 +53,27 @@
     "url": "git+https://github.com/bwinton/SnoozeTabs.git"
   },
   "scripts": {
-    "start": "npm run build && npm-run-all --parallel watch run storybook",
+    "start": "npm run build && npm-run-all --parallel test watch run storybook",
     "start-win": "npm run build && npm run watch",
```

Comment:
Let's add the test to `start-win`, too… (and change it to `npm-run-all --parallel` while we're at it).